python/cac_tripplanner/cms/models.py

```diff
@@ -5,14 +5,25 @@
 
 class Article(models.Model):
 
+    class ArticleTypes(object):
+        community_profile = 'prof'
+        tips_and_tricks = 'tips'
+
+        CHOICES = (
+            (community_profile, 'Community Profile'),
+            (tips_and_tricks, 'Tips and Tricks'),
+        )
+
     title = models.CharField(max_length=80)
     slug = models.SlugField()
     author = models.ForeignKey(User)
     teaser = RichTextField()  # above the fold
     content = RichTextField(null=True, blank=True)  # below the fold
+    publish_date = models.DateTimeField(blank=True, null=True)
     published = models.BooleanField(default=False)
```

Comment:
It would probably be better if published were removed here, and a method is_published() was added to the model, since keeping publish_date and published in sync is a pain